src/ngAnimate/animateCss.js

```diff
@@ -749,11 +747,6 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
           options.onDone();
         }
 
-        // Remove the transitionend / animationend listener(s)
-        if (events) {
-          element.off(events.join(' '), onAnimationProgress);
```

Comment:
If that is indeed the issue, I think it makes more sense to fix `jqLiteOff` instead of fixing it here.
BTW, prior to `v1.3.0-beta.19`, `jqLiteOff` did the right thing (using `isUndefined(type)` instead of `!type`), until someone ["optimized" it][1] &trade; :stuck_out_tongue: 

[1]: https://github.com/angular/angular.js/commit/abb17cce8b459e4646d1c2a2428b691c3d95fb4c